Documentation~/Cinemachine3rdPersonFollow.md

```diff
@@ -0,0 +1,75 @@
+# 3rd person follow
+
+Use Cinemachine Virtual Camera’s 3rd Person Follow to keep the camera at a constant position and distance relative to the Follow target (subject to damping controls), tracking the target’s movement and rotation.
+
+The virtual camera is bolted to a virtual rig that is attached to the target. The red rods and dots visible around the target in the scene view reflect the rig setup as well as the camera position and distance relative to the target.
+
+The rig is defined by three points: the **Shoulder Offset**, the **Vertical Arm Length**, and the **Camera Distance**.
+
+Thus, the camera position is completely controlled by the Follow target’s position and orientation, and its orientation will always match that of the Follow target.
+
+
+
+For example, a rig set up with these values:
+
+![](images/CinemachineRigInspectorValuesExample.png)
+
+
+
+Produces this rig in the Scene view:
+
+![](images/CinemachineRIgSceneView.png)
+
+A virtual rig in the Scene view, using the values in the Inspector screenshot above.
+
+A: Follow target origin point
+
+B: Shoulder Offset point
+
+C: Hand offset point
+
+- The rig pivots horizontally around the Follow target’s origin point (A), and vertically around the **Shoulder Offset** point (B).
+- An arm extends vertically from the **Shoulder Offset** point, at the end of which is another point, called the hand offset point (C).
```

Comment:
Unclear.  the arm rotate vertically, so it doesn't extend vertically if it's rotated

Reply:
Done.